torch/ao/quantization/quantize_pt2e.py

```diff
@@ -74,22 +74,31 @@ def prepare_qat_pt2e(
     torch.ops.quantized_decomposed.quantize_per_tensor.tensor,
     torch.ops.quantized_decomposed.quantize_per_channel.default,
 ]
-def _quant_node_filter(n: Node) -> bool:
+def _quant_node_constraint(n: Node) -> bool:
     """If there is any pure ops between get_attr and quantize op they will be const propagated
     e.g. get_attr(weight) -> transpose -> quantize -> dequantize*
     (Note: dequantize op is not going to be constant propagated)
 
-    Note: this filter does not handle the case when there are any additional ops after quantize:
-    e.g. get_attr(weight) -> transpose1 -> quantize -> transpose2 -> dequantize*
-    this filter will filter out the above example since the last op in the sequence and we'll not
-    do constant propagation for  the above case.
+    This filter is added because we don't want to constant fold the things that are not
+    related to quantization
     """
     return n.op == "call_function" and n.target in _QUANT_OPS
 
 def convert_pt2e(
     model: GraphModule,
     use_reference_representation: bool = False,
+    fold_quantize: bool = False,
 ) -> GraphModule:
+    """Convert a calibrated/trained model to a quantized model
+
+    Args:
+    model: calibrated/trained model
+    use_reference_representation: boolean flag to indicate whether to produce referece representation or not
+    fold_quantize: boolean flag to indicate whether fold the quantize op or not
```

Comment:
Put a note saying, this will eventually be removed, so users should prefer using fold_quantize=True for future compatibility. And a note saying, there will be a BC breaking change in future.